MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
True, and I didn't mean to imply that. There may be conflicts of interest / priority though, for people working on behalf of a company/organisation, so including this information makes this more transparent.

Not making allegations or anything :heart: , just trying to keep the information as objective / transparent as possible.